src/backend/distributed/commands/dependencies.c

```diff
@@ -55,29 +100,57 @@ static char * DropTableIfExistsCommand(Oid relationId);
  * postgres native CREATE IF NOT EXISTS, or citus helper functions.
  */
 static void
-EnsureDependenciesExistOnAllNodes(const ObjectAddress *target)
+EnsureRequiredObjectExistOnAllNodes(const ObjectAddress *target,
+									RequiredObjectSet requiredObjectSet)
 {
-	List *dependenciesWithCommands = NIL;
+	if (!IsAnyObjectDistributed(list_make1((ObjectAddress *) target)))
```

Comment:
We could use `IsObjectDistributed`

Reply:
I needed to remove this line since it caused failure